packages/edit-site/src/components/sidebar-navigation-item/index.js

```diff
@@ -34,7 +35,14 @@ export default function SidebarNavigationItem( {
 						icon={ icon }
 						size={ 24 }
 					/>
-					<FlexItem>{ children }</FlexItem>
+					<FlexBlock>{ children }</FlexBlock>
+					{ withChevron && (
+						<Icon
+							style={ { fill: 'currentcolor' } }
+							icon={ chevronRight }
+							size={ 24 }
+						/>
+					) }
```

Comment:
I wonder what if we keep this button? Maybe this will make the transition less jarring.

Reply:
What transition?

Reply:
The transition when you "drill down" which makes this button appear. It's not that important. :)

Reply:
Still not sure I understand which button you're talking about :) This component is just a reusable component used for all the menu items of the menus in all screens.

Reply:
I know, I'm just nitpicking tbh. I'm just not finding the animation that pleasant in general. Something feels weird about the sliding.

Here's what I mean with the button:

![animation](https://user-images.githubusercontent.com/4710635/206661238-bd7017d2-68ff-437e-8584-5964e4d5cb21.gif)